structopt/common/population/selections/random_selection.py

```diff
@@ -1,10 +1,7 @@
 import random as random
 from itertools import combinations
 
-def random_selection(population, fits, prob):
```

Comment:
@zhongnanxu I reverted this change because I think `prob` is supposed to be passed in. I believe the goal of this function is to randomly select pairs of individuals, where each pair has a chance to be picked with probability `prob`. We can now specify `prob` in the input file like the other functions.